frontend/src/components/Forms/MineRecordForm.js

```diff
@@ -316,6 +333,21 @@ export class MineRecordForm extends Component {
             </Form.Item>
           </Col>
         </Row>
+
+        <Row gutter={16}>
+          <Col>
+            <Form.Item>
+              <Field
+                id="status_date"
+                name="status_date"
+                label="Date of Status Change"
+                placeholder="Unknown"
```

Comment:
If this is a date field, the placeholder should be "yyyy-mm-dd"

Reply:
done